nested/dialog-window.html

```diff
@@ -60,10 +61,12 @@ <h5>Are you sure to delete the screen shot</h3>
 
 		const window = WebviewWindow.getByLabel('main')
 		const dialogWindow = getCurrent()
+		const dialogWindow2 = WebviewWindow.getByLabel('dialog'); // reference to the WebviewWindow with the label 'dialog' to set decoration to false again.
 		document.getElementById('yes').addEventListener('click', async function () {
 			try {
 				await window.emit(EVENT.PERMISSION, { 'permission': true })
 				await dialogWindow.hide()
+				await dialogWindow2.setDecorations(false)	// setting Decoration to false if user iterect with yes.
```

Comment:
i dont think it would be neceassary to set decorations to false, give it a try if it works then plz remove it